### PR TITLE
SPCTR-3063 Fix: Info-box - Removed Line height inherit CSS to resolved an issue of extra space in editor.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3063

### Screenshots

- 1st case - https://share.getcloudapp.com/E0uLrdDE
- 2nd case - https://share.getcloudapp.com/04uvO1mr
- 3rd case - https://share.getcloudapp.com/ApugL8Qm

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Info Box - Removed the line-height inherit CSS to resolved an issue with extra spacing in editor.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Info Box - Removed the line-height inherit CSS to resolve an issue with extra spacing in the editor.
```

File: readme.txt [hunk 2]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Info Box - Removed the line-height inherit CSS to resolved an issue with extra spacing in editor.
```
Comment (bsf-zanev) [inline readme.txt:170]:
```suggestion
* Fix: Info Box - Removed the line-height inherit CSS to resolve an issue with extra spacing in the editor.
```

**Review by bsf-zanev — APPROVED** [review]
Just a minor change required in the changelog entry.

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md]: Done

File: readme.txt (hunk 2 above)
Comment (AbhijitNage123) [reply readme.txt]: Done

**Review by bsf-zanev — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**